rules/cloud/azure/azure_ad_user_added_to_admin_role.yml

```diff
@@ -0,0 +1,23 @@
+title: User added to an administrator's Azure AD role
+id: ebbeb024-5b1d-4e16-9c0c-917f86c708a7
+description: Detect when an user is added to an administrator's Azure AD role
+author: Raphaël CALVET, @MetallicHack
+date: 2021/10/04
+references:
+    - https://attack.mitre.org/techniques/T1098/003/
+    - https://m365internals.com/2021/07/13/what-ive-learned-from-doing-a-year-of-cloud-forensics-in-azure-ad/
+logsource:
+    service: azure.activitylogs
+detection: 
+   selection: 
+       Operation: 'Add member to role.'
+       Workload: 'AzureActiveDirectory'
+       ModifiedProperties{}.NewValue|endswith:
+           - 'Admins'
+           - 'Administrator'
+   condition: selection
+falsepositives:
+    - PIM (Privileged Identity Management) generates this event each time 'eligible role' is enabled. 
+level: medium
+status: experimental
+tags: persistence
```

Comment:
```yaml
tags: 
    - attack.persistence
    - attack.t1098.003
```
cf: https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#tags

Reply:
Thanks for you review.
I changed the _title_ and the _tags_ fields.